### PR TITLE
Ready Player None

### PR DESCRIPTION
**Pull request by scrubmcnoob**
ready or not, ready player none is coming to theaters soon
:cl: scrubmcnoob
tweak: Playing/Connected now only shows players who are Connected:
/:cl:
why:
This can be used to metagame game modes.

### CODE REVIEW

**Comment by Gousaid67** [comment]
So wait what does this do? does it remove the player count?

**Comment by AffectedArc07** [comment]
@Gousaid67 Example:
Current:
Playing/Connected: 35/40

New:
Connected: 40

As for my stance on this, I dont exactly see why.

On a side note, the film made me think VRChat: The movie

**Comment by Gousaid67** [comment]
Yeah i dont know my stance for this, or why would you remove it in the first place, its nice to know how many players are actually active in a game.

**Comment by Cactus2388** [comment]
It can be easily metagamed. I support this change

**Comment by AdamElTablawy** [comment]
@Gousaid67 @AffectedArc07 you can pretty much guess at least two different roundtypes with this info

**Comment by scrubmcnoob** [comment]
done

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -576,7 +576,7 @@
 		stat(null, "Server Time: [time2text(world.timeofday, "YYYY-MM-DD hh:mm:ss")]")
 		stat(null, "Station Time: [worldtime2text()]")
 		stat(null, "Time Dilation: [round(SStime_track.time_dilation_current,1)]% AVG:([round(SStime_track.time_dilation_avg_fast,1)]%, [round(SStime_track.time_dilation_avg,1)]%, [round(SStime_track.time_dilation_avg_slow,1)]%)")
-		stat(null, "Playing/Connected: [get_active_player_count(0,0,0)]/[GLOB.clients.len]")
+		stat(null, "Connected: [GLOB.clients.len]")
```
Comment (SamHPurp) [inline code/modules/mob/mob.dm]: It'd be nice if this shows both to admins, and just connected to players.

**Pull request merged**